portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase

```diff
@@ -145,4 +154,22 @@ definition {
 		ACSegments.assertSegmentPresent(segmentName = "Dynamic Segment Test");
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8460 | Test Summary: Order the Segment List"
+	@ignore = "true"
+	@priority = "4"
+	test OrderSegmentList {
+
+		// TODO LRAC-8460 OrderSegmentList
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8461 | Test Summary: Paginate Segment list"
+	@ignore = "true"
+	@priority = "4"
+	test PaginateSegmentList {
+
+		// TODO LRAC-8461 PaginateSegmentList
```

Comment:
hi @AfonsoHSantos , this test is a priority should be 3 according to our test map.